squirtgun-api/src/main/java/me/lucyy/squirtgun/platform/Platform.java

```diff
@@ -102,7 +102,7 @@
 	/**
 	 * Gets a list of all online players.
 	 */
-	List<SquirtgunPlayer> getOnlinePlayers();
+	List<? extends SquirtgunPlayer> getOnlinePlayers();
```

Comment:
what's this needed for?

Reply:
So in platform impls it can return things like `List<FabricPlayer>`, `List<BukkitPlayer>` etc.

Reply:
why cant they just return a `List<SquirtgunPlayer>`? maybe im being thick but i dont get why this is necessary

Reply:
Actually this is useless, bounded wildcards are erased and compile down to just `T`